<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerContentDownloadDialog</class>
 <widget class="QDialog" name="ServerContentDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Загрузка контента сервера</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Загрузка контента сервера...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
